views/partials/top-level/remove-faq.hbs

```diff
@@ -113,62 +113,7 @@
             </div>
             <div class="remove-faq__details-container">
               <p>We scan for sites that collect and sell your personal data like
-                names, addresses, and phone numbers. Here is a list of the some
-                common sites:</p>
-              <div>
-                <ul class="remove-faq__col-3">
-                  <li>anywho.com</li>
-                  <li>backgroundalert.com</li>
-                  <li>beenverified.com</li>
-                  <li>checkpeople.com</li>
-                  <li>checkthem.com</li>
-                  <li>clustrmaps.com</li>
-                  <li>cocofinder.com</li>
-                  <li>cyberbackgroundchecks.com</li>
-                  <li>dobsearch.com</li>
-                  <li>emailsherlock.com</li>
-                  <li>familytreenow.com</li>
-                  <li>fastpeoplesearch.com</li>
-                  <li>freepeopledirectory.com</li>
-                  <li>homemetry.com</li>
-                  <li>hpccusa.com</li>
-                  <li>infotracer.com</li>
-                  <li>instantcheckmate.com</li>
-                  <li>intelius.com</li>
-                  <li>kiwisearches.com</li>
-                  <li>locatepeople.com</li>
-                  <li>mylife.com</li>
-                  <li>neighborwho.com</li>
-                  <li>nuwber.com</li>
-                  <li>peekyou.com</li>
-                  <li>peoplefinders.com</li>
-                  <li>pplcheck.com</li>
-                  <li>publicdatadigger.com</li>
-                  <li>publicdatausa.com</li>
-                  <li>publicemailrecords.com</li>
-                  <li>publicinfoservices.com</li>
-                  <li>publicrecordsnow.com</li>
-                  <li>radaris.com</li>
-                  <li>searchpeoplefree.com</li>
-                  <li>searchquarry.com</li>
-                  <li>smartbackgroundchecks.com</li>
-                  <li>socialcatfish.com</li>
-                  <li>spokeo.com</li>
-                  <li>spydialer.com</li>
-                  <li>spyfly.com</li>
-                  <li>thatsthem.com</li>
-                  <li>truthfinder.com</li>
-                  <li>unmask.com</li>
-                  <li>usphonebook.com</li>
-                  <li>ussearch.com</li>
-                  <li>veripages.com</li>
-                  <li>whitepages.com</li>
-                  <li>xlek.com</li>
-                  <li>yellowbook.com</li>
-                  <li>zabasearch.com</li>
-                  <li>peoplebyname.com</li>
-                </ul>
-              </div>
+                names, addresses, and phone numbers. While the list changes frequently, you can <a href="https://www.thekanary.com/faq" target="_blank">visit Kanary's FAQ page</a> to see the most common sites.</p>
```

Comment:
Q: Should this also have `rel="noopener noreferrer"` attribute since it's an external site?